quipucords/scanner/vcenter/inspect.py

```diff
@@ -199,13 +201,17 @@ def parse_vm_props(self, props, host_dict):
             elif prop.name == 'summary.config.uuid':
                 facts['vm.uuid'] = prop.val
             elif prop.name == 'runtime.host':
-                host_facts = host_dict[str(prop.val)]
-                facts['vm.host.name'] = host_facts['host.name']
-                facts['vm.host.cpu_cores'] = host_facts['host.cpu_cores']
-                facts['vm.host.cpu_count'] = host_facts['host.cpu_count']
-                facts['vm.host.cpu_threads'] = host_facts['host.cpu_threads']
-                facts['vm.cluster'] = host_facts['host.cluster']
-                facts['vm.datacenter'] = host_facts['host.datacenter']
+                if host_dict.get(str(prop.val)):
```

Comment:
if you want to check if a key is on a dict, a more pythonic way would be `if str(prop.val) in host_dict`

Reply:
Or, since you need the value below you could do something like:

```python
host_facts = host_dict.get(str(prop.val))
if host_facts:
   # from line 206 and on here
```